RLTest/env.py

```diff
@@ -152,7 +152,7 @@ def getKwargs(self):
 class Env:
     RTestInstance = None
     EnvCompareParams = ['module', 'moduleArgs', 'env', 'useSlaves', 'shardsCount', 'useAof',
-                        'useRdbPreamble', 'forceTcp']
+                        'useRdbPreamble', 'forceTcp', 'decodeResponses']
```

Comment:
I do not think it should be part of the env definition, we could make it argument to env.cmd/env.expect/env.getConnection .. WDYT?